app/models/announcement.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Announcement < ApplicationRecord
+  def self.current
+    order("created_at desc").first || new
+  end
+
+  def exists?
```

Comment:
there is a method called `persisted?` that does this

Reply:
ohk will use it instead of new_record?